monkey/monkey_island/cc/ui/src/components/pages/LandingPage.tsx

```diff
@@ -74,4 +89,41 @@ function MonkeyInfo() {
   );
 }
 
+function ScenarioInfo() {
+    // TODO change links when added to documentation
+    return (
+      <>
+        <div className={'scenario-info'}>
+          Check monkey documentation hub for better explanation
+          on <a href='https://www.guardicore.com/infectionmonkey/docs' rel="noopener noreferrer" target="_blank">
+           Ransomware
+          </a> and <a href='https://www.guardicore.com/infectionmonkey/docs' rel="noopener noreferrer" target="_blank">
+           Custom
+          </a> simulations.
+        </div>
+      </>
+    );
+}
+
+function GuardicoreLogo() {
```

Comment:
This code is duplicated from `SideNavComponent.tsx`. SideNavComponent should be modified to use this component instead of duplicating the contents. That probably means this needs to go into its own file.